src/main/resources/application.yml

```diff
@@ -50,7 +50,7 @@ spring:
     generate-ddl: true
     show-sql: false
     hibernate:
-      ddl-auto: create-drop
+      ddl-auto: update
```

Comment:
Could we please revert the default application.yml changes?

Reply:
Reverted